grml2usb

```diff
@@ -513,19 +523,19 @@ def check_boot_flag(device):
         )
 
 
-def mkfs_fat16(device):
-    """Format specified device with VFAT/FAT16 filesystem.
+def mkfs_vfat(device):
+    """Format specified device with VFAT filesystem.
 
     @device: partition that should be formated"""
     assert options is not None
 
     if options.dryrun:
-        logging.info("Would execute mkfs.vfat -F 16 %s now.", device)
+        logging.info("Would execute mkfs.vfat %s now.", device)
         return 0
 
-    logging.info("Formating partition with fat16 filesystem")
-    logging.debug("mkfs.vfat -F 16 %s", device)
-    proc = subprocess.Popen(["mkfs.vfat", "-n", "GRML", "-F", "16", device])
+    logging.info("Formating partition with vfat filesystem")
```

Comment:
still says vfat here. so either

```suggestion
    logging.info("Formatting partition with FAT filesystem")
```
or
```suggestion
    logging.info("Formatting partition using mkfs.vfat")
```

(Would be smart to also include the partition name, but out of scope)